hid-ft260.c

```diff
@@ -671,6 +681,13 @@ static int ft260_smbus_xfer(struct i2c_adapter *adapter, u16 addr, u16 flags,
 		return ret;
 	}
 
+	if (time_is_before_jiffies(dev->need_wakeup_at)) {
+		ft260_dbg("sending status request to wakeup device");
+		ret = ft260_xfer_status(dev);
+	}
+
```

Comment:
I would move it into `ft260_smbus_write()` since `ft260_smbus_read()` works fine upon exiting from power-saving mode.